EngineLayer/Psm.cs

```diff
@@ -401,14 +401,14 @@ internal void Add(Psm psmParent)
         private Tuple<string, double?> Resolve(IEnumerable<double> enumerable)
         {
             double doubleTol = 1e-6;
-            var list = enumerable.ToList();
+            var list = enumerable.Distinct().ToList();
```

Comment:
and I think the delimiter should be one character; Excel does allow more than one character when splitting a column into multiple columns using a delimiter